### ISSUE TITLE
Please add to stable repository if shown working for users

### ISSUE THREAD

**Issue by Apollon77** [UncleSamSwiss/ioBroker.i2c#8]

**Comment by UncleSamSwiss** [comment]
There are some issues (one PR I'm not sure if it is OK --> to be verified) and some more device I would like to add before making this adapter official.
I ordered some hardware and will be adding the devices in the coming months.

**Comment by UncleSamSwiss** [comment]
Depends on #7, #10, #11, #12, #13

**Comment by UncleSamSwiss** [comment]
Duplicate of #39
